packages/page-staking/src/Validators/Summary.tsx

```diff
@@ -16,28 +15,22 @@ import { useTranslation } from '../translate';
 interface Props {
   className?: string;
   nominators?: string[];
-  stakingOverview?: DeriveStakingOverview;
   targets: SortedTargets;
+  eraValidators: EraValidators;
 }
 
-function Summary ({ className = '', stakingOverview, targets: { counterForNominators, inflation: { idealStake, inflation, stakedFraction }, nominators, waitingIds } }: Props): React.ReactElement<Props> {
+function Summary ({ className = '', eraValidators, targets: { counterForNominators, inflation: { idealStake, inflation, stakedFraction }, nominators } }: Props): React.ReactElement<Props> {
   const { t } = useTranslation();
 
   return (
     <SummaryBox className={className}>
       <section>
-        <CardSummary label={t<string>('validators')}>
-          {stakingOverview
-            ? <>{formatNumber(stakingOverview.validators.length)}&nbsp;/&nbsp;{formatNumber(stakingOverview.validatorCount)}</>
-            : <Spinner noLabel />
-          }
-        </CardSummary>
         <CardSummary
           className='media--900'
-          label={t<string>('waiting')}
+          label={t<string>('era validators')}
```

Comment:
I'm wondering -- would it be easy to get rid of "ideal staked 50.0%" on this page as well. That's perhaps the last confusing element on this page...

Reply:
Sure, done